backend/substrapp/tasks/tasks.py

```diff
@@ -117,10 +117,11 @@ def _put_model(subtuple, subtuple_directory, model_content, model_hash, traintup
 
         if not os.path.exists(model_dst_path):
             os.link(model.file.path, model_dst_path)
-        else:
-            # verify that local subtuple model file is not corrupted
-            if get_hash(model_dst_path, traintuple_key) != model_hash:
-                raise Exception('Model Hash in Subtuple is not the same as in local medias')
+
+    finally:
```

Comment:
this should not be updated

Reply:
I found it better to check that the model we put is the wanted one in all case.
Is there any issue doing that ?

Reply:
I don't know if it implies an issue but was not in the original workflow.
I'm not the right person for reviewing this change.
We need someone else help on that

Reply:
Oh, you're right, the workflow will change by doing that.
@camillemarini is it an issue to do the systematic check or I need to revert to the previous workflow ? (check only on download from external node)

Reply:
Solved by reverting this modification.